lua/telescope/actions/generate.lua

```diff
@@ -53,4 +53,9 @@ action_generate.which_key = function(opts)
   end
 end
 
+action_generate.copy_command_to_reg = function(opts)
+  return function(prompt_bufnr)
+    actions.copy_command_to_reg(prompt_bufnr, opts)
+  end
+end
```

Comment:
Small nit: maybe an extra line here before return :)

Reply:
The formatter removes the extra line :disappointed:, not sure how to deal with it yet.